src/pages/Main/PostList/PostList.js

```diff
@@ -0,0 +1,67 @@
+import styled from 'styled-components';
+import { setFlex, theme } from '../../../styles/theme';
+function Post(post) {
+  return (
+    <PostCard>
+      <PostImageWrap>
+        <img alt="company" src={post.post.img} />
+      </PostImageWrap>
+      <CompanySummery>
+        <h2>{post.post.title}</h2>
+        <p>{post.post.company}</p>
+        <p className="pTagGray">{post.post.location}</p>
+        <p className="pTagGray">{post.post.reward}</p>
```

Comment:
```suggestion
        <Text color="gray">{post.location}</p>
        <Text color="gray">{post.reward}</p>
```
- props는 `const { post } = props`와 같은 형태로 구조분해 할당 해서 쓰시면 됩니다
- Text를 스타일드 컴포넌트로 만들고 props로 color를 받게 하는 건 어떨까요?

Reply:
아직 익숙치가 않내요 ㅠㅠ 참고해서 수정 해 보겠습니다!